### PR TITLE
feat: add namespace support

### PR DESCRIPTION
**Pull request by XGHeaven**
RFC: SVG

This PR is the dep of the SVG supports.

Changes:

- [x] Add `createElementNS`.
- [x] Add `Element.prototype.namespaceURI`.
- [x] Add `Element.prototype.prefix`
- [x] Add `Element.prototype.localName`
- [x] Add test for `createElementNS`.
- [x] Use `prefix` and `qualified_name` to replace `tag_name` in `node.cc`

### CODE REVIEW

File: bridge/core/dom/document.cc [hunk 1]
```diff
@@ -52,7 +52,7 @@ Element* Document::createElement(const AtomicString& name, ExceptionState& excep
     return MakeGarbageCollected<WidgetElement>(name, this);
   }
 
-  return MakeGarbageCollected<HTMLUnknownElement>(name, *this);
```
Comment (andycall) [inline bridge/core/dom/document.cc:55]: Why remove this?

Comment (XGHeaven) [reply bridge/core/dom/document.cc:55]: Sorry, I misunderstand with spec. I will add it back.

**Review by andycall — APPROVED** [review]

File: bridge/core/dom/element.cc [hunk 2]
```diff
@@ -351,7 +374,8 @@ ScriptPromise Element::toBlob(double device_pixel_ratio, ExceptionState& excepti
 }
 
 std::string Element::outerHTML() {
-  std::string s = "<" + nodeNameLowerCase();
+  std::string tagname = local_name_.ToStdString(ctx());
```
Comment (cnryb) [inline bridge/core/dom/element.cc:377]: No need to convert to lowercase here anymore?

File: webf/lib/src/launcher/controller.dart [hunk 3]
```diff
@@ -441,6 +441,18 @@ class WebFViewController implements WidgetsBindingObserver, ElementsBindingObser
     }
   }
 
+  void createElementNS(int targetId, Pointer<NativeBindingObject> nativePtr, String uri, String tagName) {
+    if (kProfileMode) {
+      PerformanceTiming.instance().mark(PERF_CREATE_ELEMENT_START, uniqueId: targetId);
+    }
+    assert(!_existsTarget(targetId), 'ERROR: Can not create element with same id "$targetId"');
```
Comment (cnryb) [inline webf/lib/src/launcher/controller.dart:448]: Maybe it is better to put it on the first line, in this function.

**Review by devjiangzhou — APPROVED** [review]

File: bridge/core/dom/element.cc (hunk 2 above)
Comment (andycall) [reply bridge/core/dom/element.cc:377]: name are tolowercased at createElement() function.

https://github.com/openwebf/webf/pull/126/files#diff-21f9bbf31500c133b00f2df750d9a69f2532cbd12a7eba8c9310f7bb22d2c2f0R43

**Pull request merged**